src/ui.text-file/configs/default.config.js

```diff
@@ -1,15 +1,20 @@
 export default /*tw*/ {
   file: "",
-  info: "flex items-start",
+  info: "flex items-center gap-2 ",
   icon: "mr-3",
   iconName: "description",
-  itemImage: "rounded-sm",
-  text: {
-    base: "whitespace-normal pt-0.5 hover:underline hover:decoration-dashed hover:underline-offset-4",
+  itemImage: "rounded-sm max-w-7",
+  itemLabel: {
```

Comment:
do we have just `label` key here? If not it's better to change the key name. same for `itemImage`